Server/ReasnAPI/ReasnAPI/Services/InterestService.cs

```diff
@@ -56,6 +56,21 @@ public void DeleteInterest(int id)
         context.SaveChanges();
     }
 
+    public void ForceDeleteInterest(int id)

```

Comment:
So if you do it this way, we would need to have also this kind of method for tags and parameters